lib/testing-helpcenter/help_center_tags/articles_with_tag_popup.rb

```diff
@@ -8,10 +8,23 @@ def initialize(instance)
     super(instance.webdriver.driver)
     @instance = instance
     @pop_up_tags_xpath = "//div[@id = 'textHelpContent']//a"
+    @close_button_xpath = "//div[@id = 'textHelpContainer']//a[contains(@class, 'closeContainerButton')]"
```

Comment:
You can store `//div[@id = 'textHelpContent']` as `base_xpath` variable and make all other xpath based on this var